analytics_data_api/v0/views/learners.py

```diff
@@ -28,7 +28,7 @@
 
 class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
     """
-    Get a particular student's data for a particular course.
+    Get data for a particular learner in a particular course.
```

Comment:
What term are we using? Learner or user? I see both in this doc (I definitely contributed to this trend :laughing:)